src/qt_gui/translations/pl_PL.ts

```diff
@@ -605,6 +605,16 @@
             <source>Game already installed\n%1\nWould you like to overwrite?</source>
             <translation>Gra już zainstalowana\n%1\nCzy chcesz ją nadpisać?</translation>
         </message>
+        <message>
+            <location filename="../main_window.cpp" line="673"/>
+            <source>DLC Installation</source>
+            <translation>DLC Installation</translation>
+        </message>
+        <message>
+            <location filename="../main_window.cpp" line="674"/>
+            <source>Would you like to install DLC: %1?</source>
+            <translation>Would you like to install DLC: %1?</translation>
+        </message>
```

Comment:
Use for all new strings, this should be better approach, ig
You can test it out too:
```suggestion
        <message>
            <location filename="../main_window.cpp" line="674"/>
            <source>Would you like to install DLC: %1?</source>
            <translation type="unfinished"></translation>
        </message>
```

Reply:
It's not a better approach. Regular users don't use Linguist.
Imagine if someone who wants to translate something. opens this file and sees `<translation type="unfinished"></translation>` and they like "wtf is this, what is type, what is unfinished, how do I translate". There're no projects which use this approach

Reply:
Check DuckStation or PCSX2 translation files.
PCSX2 is done through Crowdin so it's easier.
DuckStation is PR based.

Reply:
Crowdin is good in Pcsx2. I would suggest using this.